src/main/resources/templates/java/test/testFiles/behavior/BehaviorTest.java

```diff
@@ -113,6 +161,7 @@ protected Object valueForAttribute(Object object, String attributeName) {
      * @return The wanted method.
      */
     protected Method getMethod(Object object, String methodName, Class<?>... parameterTypes) {
+        requireNonNull(object, "receiver must not be null");
```

Comment:
I don't think that `requireNonNull`. How would the feedback in Artemis look like that students get to see in such a case? I think we should rather provide a good fail message.